opentracing/scope.py

```diff
@@ -78,5 +78,5 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         and added as a tag to the :class:`Span`.
         :attr:`~operation.ext.tags.ERROR` will also be set to `True`.
         """
-        self.span._on_error(exc_type, exc_val, exc_tb)
+        Span._on_error(self.span, exc_type, exc_val, exc_tb)
```

Comment:
@yurishkuro
In terms of linters, I want to rename `_on_error` to `on_error`, since `PyCharm`'s linter justly complains of access to a protected member here.
What do you think?

Reply:
private is better, less API surface

Reply:
ok, got it.